lang/el/index.md

```diff
@@ -0,0 +1,409 @@
+τους χρήστες σας, χρησιμοποιήστε τον αριθμό έκδοσης για να τους ενημερώσετε.
+
+### Πώς πρέπει να χειριστώ τη λειτουργικότητα που καταργείται;
+
+Η κατάργηση υπάρχουσας λειτουργικότητας είναι φυσιολογικό μέρος της ανάπτυξης
+λογισμικού και συχνά απαιτείται για να σημειωθεί πρόοδος. Όταν καταργείτε μέρος
+του δημόσιου API σας, θα πρέπει να κάνετε δύο πράγματα: (1) να ενημερώσετε την
+τεκμηρίωσή για να μάθουν οι χρήστες σχετικά με την αλλαγή, (2) να εκδώσετε μια
+νέα δευτερεύουσα κυκλοφορία με την κατάργηση. Προτού αφαιρέσετε εντελώς τη
+λειτουργικότητα σε μια νέα κύρια κυκλοφορία, θα πρέπει να υπάρχει τουλάχιστον
+μία δευτερεύουσα κυκλοφορία που να περιέχει την κατάργηση, ώστε οι χρήστες να
+μπορούν να μεταβούν ομαλά στο νέο API.
+
+### Έχει το SemVer όριο μεγέθους στη συμβολοσειρά έκδοσης;
+
+Όχι, αλλά χρησιμοποιήστε καλή κρίση. Μια συμβολοσειρά έκδοσης 255 χαρακτήρων
+για παράδειγμα, είναι πιθανώς υπερβολική. Επίσης, συγκεκριμένα συστήματα μπορεί
+να επιβάλλουν τα δικά τους όρια στο μέγεθος της συμβολοσειράς.
+
+### Είναι το «v1.2.3» μια σημασιολογική έκδοση;
+
+Όχι, το «v1.2.3» δεν είναι σημασιολογική έκδοση. Ωστόσο, το πρόθεμα μιας
+σημασιολογικής έκδοσης με ένα «v», είναι ένας συνηθισμένος τρόπος (στα Αγγλικά)
+για να δηλωθεί ότι είναι αριθμός έκδοσης. Η συντομογραφία του «version» ως «v»
+εμφανίζεται συχνά με τον έλεγχο εκδόσεων. Παράδειγμα:
+`git tag v1.2.3 -m "Έκδοση κυκλοφορίας 1.2.3"`, στην περίπτωση αυτή το «v1.2.3»
+είναι το όνομα μιας ετικέτας και η σημασιολογική έκδοση είναι «1.2.3».
+
+### Υπάρχει μια προτεινόμενη κανονική έκφραση (RegEx) για τον έλεγχο μιας συμβολοσειράς SemVer;
+
+Υπάρχουν δύο. Μία με ονοματισμένες ομάδες, named groups, για εκείνα τα συστήματα
+που τα υποστηρίζουν (PCRE [Perl Compatible Regular Expressions, δηλ. Perl, PHP και R], Python και Go).
+
+Δείτε: <https://regex101.com/r/Ly7O1x/3/>
+
+```
+^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+```
+
+Και μία με αριθμημένες ομάδες σύληψης, capture groups, (έτσι cg1 = κύρια,
+cg2 = δευτερεύουσα, cg3 = διορθωτική, cg4 = προκυκλοφορία και 
+cg5 = μεταδεδομέναχτισίματος) που είναι συμβατή με ECMA Script (JavaScript),
+PCRE (Perl Compatible Regular Expressions, δηλαδή Perl, PHP και R), Python και Go.
+
+Δείτε: <https://regex101.com/r/vkijKf/1/>
+
+```
+^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+```
+
+Περί
+-----
+
+Η προδιαγραφή Σημασιολογικής Δημιουργίας Εκδόσεων συγγράφηκε αρχικά από
+τον [Tom Preston-Werner](https://tom.preston-werner.com), δημιουργό του Gravatar
+και συνιδρυτή του GitHub.
+
+Την αρχική μετάφραση στα Ελληνικά έκανε ο [Ευάγγελος Σκαρμούτσος](https://skarmoutsosv.gr)
```

Comment:
Could you replace it with your GitHub user profile URL? I'd like to avoid having external links as much as possible.